cpp/cmake/Dependencies.cmake

```diff
@@ -38,8 +38,8 @@ else(DEFINED ENV{RAFT_PATH})
   set(RAFT_DIR ${CMAKE_CURRENT_BINARY_DIR}/raft CACHE STRING "Path to RAFT repo")
 
   ExternalProject_Add(raft
-    GIT_REPOSITORY    https://github.com/rapidsai/raft.git
-    GIT_TAG           fc46618d76d70710b07d445e79d3e07dea6cad2f
+    GIT_REPOSITORY    https://github.com/cjnolet/raft.git
+    GIT_TAG           7bffddfe69aaa370d2affb2b1bb4bf7735589c1f
```

Comment:
Just adding comment to track: needs to be changed back to main raft after merge of https://github.com/rapidsai/raft/pull/181 (right?)